### PR TITLE
Issue #17: CPQ Script translation

### PR DESCRIPTION
**Pull request by Evol93**
Change text to English and add Map in use check

### CODE REVIEW

File: scripts/npc/2042005.js [hunk 1]
```diff
@@ -45,15 +45,17 @@ function action(mode, type, selection) {
...
                     status = 10;
-                    cm.sendOk("Certifique se todos em seu grupo est�o dentre os n�veis corretos (" + cpqMinLvl + "~" + cpqMaxLvl + ")!");
+                    cm.sendOk("Make sure everyone in your party is among the correct levels (" + cpqMinLvl + "~" + cpqMaxLvl + ")!");
                 } else if (isOutMap > 0) {
                     status = 10;
-                    cm.sendOk("Existe algu�m do grupo que n�o esta no mapa!");
+                    cm.sendOk("There are some of the party members that is not on the map!");
                 } else {
-                    cm.sendCPQMapLists2();
+                    if (!cm.sendCPQMapLists2()) {
```
Comment (minimucho1) [inline scripts/npc/2042005.js]: What is this condition checking for? `sendCPQMapLists2` always returns `void`

**Review by minimucho1 — CHANGES_REQUESTED** [review]

File: scripts/npc/2042005.js (hunk 1 above)
Comment (minimucho1) [reply scripts/npc/2042005.js]: There is also language referenced in `NPCConversationManager.java` that uses `LanguageConstants.java` to write the final message. We need to ensure that the final message is also using the correct localization.

**Review by minimucho1 — DISMISSED** [review]
Looks good.

**Comment by minimucho1** [comment]
The only thing left is to amend these commits with the proper issue number before we merge. Waiting on @Evol93

**Comment by Evol93** [comment]
That should be it.

**Comment by minimucho1** [comment]
This is not following the commit message format. I can't merge this change in.

**Comment by minimucho1** [comment]
This PR has 29 commits that have multiple duplicates. We need to squash these commits and properly rebase them before merging this change in.

**Review by minimucho1 — CHANGES_REQUESTED** [review]
Please review my comments

**Comment by Evol93** [comment]
Ugh...how or what am i suppose to do. I dont use github.

**Pull request merged**